readme.md

```diff
@@ -111,13 +111,15 @@ A normalized string representing the entire address that was passed into the con
 ## Development
 
 1. create conda environment
-   - `conda create --clone arcgispro-py3 --name sweeper`
+    - `conda create --name sweeper python=3.7`
```

Comment:
**thought**: python 3.9 is the latest version and should be what we recommend

Reply:
Does arcpy work with 3.9?

Reply:
Good question. I would assume yes.

Reply:
Just as a heads up, I just got this warning in one of my projects (latest arcpy, python 3.7.9):

`c:\users\jdadams\miniconda3\envs\auditor\lib\site-packages\arcgis\_impl\common\_mixins.py:8: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3,and in 3.9 it will stop working`

So arcpy on 3.9 may not work consistently.